examples/websocket_client.py

```
@@ -0,0 +1,281 @@
+            
+            if response_data.get("event") == "phx_reply" and response_data.get("payload", {}).get("status") == "ok":
+                payload = response_data["payload"]["response"]
+                self.subscription_id = payload.get("subscription_id")
+                logger.info(f"✅ Connected to WandererKills WebSocket")
+                logger.info(f"📋 Subscription ID: {self.subscription_id}")
+                
+                # Start listening for messages
+                asyncio.create_task(self._listen_for_messages())
+                
+                return payload
+            else:
```

Ruff (0.11.9)

Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)